### PR TITLE
[zh-cn] Resync Windows debug page

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

File: content/zh-cn/docs/tasks/debug/debug-cluster/windows.md [hunk 1]
```diff
@@ -84,23 +109,7 @@ content_type: concept
...
-   Windows Pod 没有为 ICMP 协议编写出站规则，但 TCP/UDP 是支持的。当试图演示与集群外部资源的连接时，可以把 `ping <IP>` 替换为 `curl <IP>` 命令。
-
-   如果你仍然遇到问题，很可能你需要额外关注
-   [cni.conf](https://github.com/Microsoft/SDN/blob/master/Kubernetes/flannel/l2bridge/cni/config/cni.conf)
-   的配置。你可以随时编辑这个静态文件。更新配置将应用于新的 Kubernetes 资源。
-
+   -->
    Kubernetes 的网络需求之一 (查看 [Kubernetes 模型](/zh-cn/docs/concepts/cluster-administration/networking/)) 
    是集群通信不需要内部的 NAT。
    为了遵守这一要求， 对于你不希望发生的出站 NAT 通信，这里有一个
```
Comment (Sea-n) [inline content/zh-cn/docs/tasks/debug/debug-cluster/windows.md:115]:
```suggestion
   为了遵守这一要求，对于你不希望发生的出站 NAT 通信，这里有一个
```

File: content/zh-cn/docs/tasks/debug/debug-cluster/windows.md [hunk 2]
```diff
@@ -246,43 +256,39 @@ content_type: concept
...
    A workaround is to relaunch `start.ps1` or relaunch it manually as follows:
-
-   ```powershell
-   [Environment]::SetEnvironmentVariable("NODE_NAME", "<Windows_Worker_Hostname>")
-   C:\flannel\flanneld.exe --kubeconfig-file=c:\k\config --iface=<Windows_Worker_Node_IP> --ip-masq=1 --kube-subnet-mgr=1
-   ```
 -->
 2. Flanneld 卡在 "Waiting for the Network to be created"
 
    关于这个[问题](https://github.com/coreos/flannel/issues/1066)有很多报告 ；
```
Comment (Sea-n) [inline content/zh-cn/docs/tasks/debug/debug-cluster/windows.md:269]:
```suggestion
   关于这个[问题](https://github.com/coreos/flannel/issues/1066)有很多报告；
```

**Review by Sea-n — COMMENTED** [review]
/lgtm

**Review by Sea-n — COMMENTED** [review]
Thanks for fixing the non-critical nits 😅

/lgtm

File: content/zh-cn/docs/tasks/debug/debug-cluster/windows.md [hunk 3]
```diff
@@ -17,37 +25,43 @@ content_type: concept
...
-   Ensure that your Pod is getting scheduled to a [compatable](https://docs.microsoft.com/virtualization/windowscontainers/deploy-containers/version-compatibility) Windows Node.
+   Ensure that your Pod is getting scheduled to a
+   [compatible](https://docs.microsoft.com/virtualization/windowscontainers/deploy-containers/version-compatibility)
+   Windows Node.
 
-   More information on how to specify a compatable node for your Pod can be found in [this guide](/docs/setup/production-environment/windows/user-guide-windows-containers/#ensuring-os-specific-workloads-land-on-the-appropriate-container-host).
+   More information on how to specify a compatible node for your Pod can be found in
+   [this guide](/docs/setup/production-environment/windows/user-guide-windows-containers/#ensuring-os-specific-workloads-land-on-the-appropriate-container-host).
 -->
 2. 我的 pod 状态显示 'ErrImgPull' 或者 ‘ImagePullBackOff’
```
Comment (howieyuen) [inline content/zh-cn/docs/tasks/debug/debug-cluster/windows.md:56]:
```suggestion
2. 我的 Pod 状态显示 'ErrImgPull' 或者 ‘ImagePullBackOff’
```

File: content/zh-cn/docs/tasks/debug/debug-cluster/windows.md [hunk 4]
```diff
@@ -84,90 +109,76 @@ content_type: concept
...
    it, users need to pass the hostname to kube-proxy as follows:
-
-   ```powershell
-   C:\k\kube-proxy.exe --hostname-override=$(hostname)
-   ```
 -->
-3. 我的 Windows 节点无法访问 `NodePort` 类型服务
-
-   从节点本身访问本地 NodePort 失败，是一个已知的限制。你可以从其他节点或外部客户端正常访问 NodePort。
+4. 容器的 vnic 和 HNS 端点正在被删除
```
Comment (howieyuen) [inline content/zh-cn/docs/tasks/debug/debug-cluster/windows.md:156]:
```suggestion
4. 容器的 VNIC 和 HNS 端点正在被删除
```

**Pull request merged**